environments/aks/dev.tfvars

```
@@ -13,6 +13,7 @@ kubernetes_cluster_agent_max_count = "10"
 
 linux_node_pool = {
   max_pods = 50
+  max_count = 20
```

as said in the other PR 👅 
https://github.com/hmcts/aks-sds-deploy/blob/67d0c2ce3e53e4edc0a76ac6819210acf1b11836/components/aks/aks.tf#L29
```suggestion
  max_nodes = 20
```